Integrations/GoogleKeyManagementService/GoogleKeyManagementService.py

```diff
@@ -0,0 +1,854 @@
+    crypto_key_version = args.get('crypto_key_version')
+
+    if crypto_key_version == 'default':
+        # if no CryptoKeyVersion given extract the primary CryptoKeyVersion.
+        crypto_key_version_name = get_primary_key_version(project_id, location_id, key_ring_id, crypto_key_id, client)
+
+    else:
+        # Construct the resource name of the CryptoKeyVersion.
+        crypto_key_version_name = client.kms_client.crypto_key_version_path(project_id, location_id, key_ring_id,
+                                                                            crypto_key_id, crypto_key_version)
+
+    # if not CryptoKeyVersion is given nor was is extracted from the CryptoKey - raise error to the user.
+    if crypto_key_version_name is None or len(crypto_key_version_name) == 0:
+        raise Exception("Please insert primary CryptoKeyVersion ID")
+
+    # Use the KMS API to disable the CryptoKeyVersion.
+    new_state = enums.CryptoKeyVersion.CryptoKeyVersionState.DISABLED
+    version = {'name': crypto_key_version_name, 'state': new_state}
+    update_mask = {'paths': ["state"]}
+
+    # Print results
+    response = client.kms_client.update_crypto_key_version(version, update_mask)
+    demisto.results(f'CryptoKeyVersion {crypto_key_version_name}\'s state has been set to '
+                    f'{enums.CryptoKeyVersion.CryptoKeyVersionState(response.state).name}.')
+
+
+def enable_key_command(client: Client, args: Dict[str, Any]) -> None:
+    """Enable a CryptoKeyVersion.
+
+    Args:
+        client(Client): User Client.
+        args(Dict): Demisto arguments.
+    """
+    project_id, location_id, key_ring_id, crypto_key_id = demisto_args_extract(client, args)
+    crypto_key_version = args.get('crypto_key_version')
+
+    if crypto_key_version == 'default':
+        # if no CryptoKeyVersion given extract the primary CryptoKeyVersion.
+        crypto_key_version_name = get_primary_key_version(project_id, location_id, key_ring_id, crypto_key_id, client)
+
+    else:
+        # Construct the resource name of the CryptoKeyVersion.
+        crypto_key_version_name = client.kms_client.crypto_key_version_path(project_id, location_id, key_ring_id,
+                                                                            crypto_key_id, crypto_key_version)
+
+    # if not CryptoKeyVersion is given nor was is extracted from the CryptoKey - raise error to the user.
+    if crypto_key_version_name is None or len(crypto_key_version_name) == 0:
+        raise Exception("Please insert primary CryptoKeyVersion ID")
+
+    # Use the KMS API to enable the CryptoKeyVersion.
+    new_state = enums.CryptoKeyVersion.CryptoKeyVersionState.ENABLED
+    version = {'name': crypto_key_version_name, 'state': new_state}
+    update_mask = {'paths': ["state"]}
+
+    # Print results
+    response = client.kms_client.update_crypto_key_version(version, update_mask)
+    demisto.results(f'CryptoKeyVersion {crypto_key_version_name}\'s state has been set to '
```

Comment:
this should be a `return`, not a `demisto.results`.
same for everywhere.
use `return (<string>, )` or `return (<string>, None, None)`

Reply:
done